src/pages/studyView/queryStudyAnalytics.ts

```diff
@@ -0,0 +1,24 @@
+import * as request from "superagent";
+
+export interface IAnalyticsResponse {
+    totalUses: number;
+    totalUniqueUses: number;
+    timesVisited: number;
+    timesUniqueVisited: number;
+}
+
+export interface IQueryResponse extends request.Response {
```

Comment:
Too general I think.    Probably you could get away with just typing this inline since it's unlikely to be used elsewhere.